deluge/ui/gtkui/statusbar.py

```diff
@@ -326,6 +331,12 @@ def _on_max_upload_speed(self, max_upload_speed):
         self.max_upload_speed = max_upload_speed
         self.update_upload_label()
 
+    def _on_get_external_ip(self, external_ip):
+        if external_ip:
```

Comment:
Maybe this instead to reduce repetition?
```
e = external_ip if external_ip else "n/a"
self.external_ip_item.set_markup(_("<b>IP</b> %s") % e)
```

Also, should we be concatenating the strings to ensure translation can happen?
```
_("<b>IP</b> " + e)
```